packages/dart/npt_flutter/lib/features/onboarding/widgets/onboarding_button.dart

```diff
@@ -129,6 +131,9 @@ class _OnboardingButtonState extends State<OnboardingButton> {
   }
 
   Future<void> onboard({required String atsign, required String rootDomain, bool isFromInitState = false}) async {
+    SyncServiceImpl.queueSize = 1;
+    SyncServiceImpl.syncRequestThreshold = 1;
+    SyncServiceImpl.syncRequestTriggerInSeconds = 1;
```

Comment:
When we get a statsNotification that commitId has increased, we want to have the sync execute as soon as possible.

Dependence on sync for dynamic addition / removal of profiles is unlikely be part of the eventual solution.